Docs/conf.py

```diff
@@ -207,3 +213,69 @@
     import sphinx_rtd_theme
     html_theme = 'sphinx_rtd_theme'
     html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+
+# -- Convert CLI module desriptions into markdown files ----------------
+
+# Each CLI module descriptor XML file is converted to two markdown files
+# in _moduledescriptions subfolder: *Overview.md and *Parameters.md.
+# Overview file contains the module title and description.
+# Parameters file contains detailed description of module inputs and
+# outputs, contributors, and acknowledgements.
+#
+# These md files are included at the top and bottom of each CLI module
+# documentation file (user_guide\modules\*.md). Custom content, such as
+# tutorials, screenshots, etc. can be added in between these includes.
+#
+# A copy of all CLI module descriptor XML files for modules that are
+# not part of the Slicer repository (e.g., BRAINS toolkit) are stored in
+# _extracli subfolder. Content of this folder must be updated manually
+# whenever they are updated in the original location. The process could
+# be automated with some effort, but since these bundled libraries do
+# not change frequently, manual update takes less work overall.
```

Comment:
> re: automation [...]  do not change frequently, manual update takes less work overall

Agreed. I suggest to add a comment to `SuperBuild.cmake` with the bash  command to run. See my previous suggestion.

Reply:
I would rather focus on implementing a good final solution rather than making the current workaround a bit more convenient.

Reply:
> I would rather focus on implementing a good final solution rather than making the current workaround a bit more convenient.

Make sense. Then, let's create an issue to track the suggestion about json description file you described above ?

Reply:
OK!

Reply:
>  let's create an issue to track the suggestion about json description file you described above ?

See https://github.com/Slicer/Slicer/issues/5669